src/electionguard/discrete_log.py

```diff
@@ -6,16 +6,39 @@
 
 from .constants import get_generator
 from .singleton import Singleton
-from .group import ElementModP, ONE_MOD_P, mult_p
+from .group import BaseElement, ElementModP, ONE_MOD_P, mult_p
 
-DLogCache = Dict[ElementModP, int]
-DLOG_MAX = 100_000_000
-"""The max number to calculate.  This value is used to stop a race condition."""
+DiscreteLogCache = Dict[ElementModP, int]
+
+_DLOG_MAX_EXPONENT = 100_000_000
```

Comment:
This now becomes a backup to the backup. The first and foremost one can be set by using the ElectionGuard config. I expected lines like:

```py
DiscreteLog().set_max_exponent(config.max_votes)
DiscreteLog().precompute_cache()
```